students/Dennis_Coffey/lesson07/html_render.py

```diff
@@ -0,0 +1,163 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Nov 29 21:13:15 2018
+
+@author: dennis
+"""
+
+"""HTML renderer for Lesson 07"""
+
+# Base element class
+class Element():
+    tag = '' # Element type
+    indent_size = ' '* 4 # Indent of tag
+#    indent_size = ''* 4 # Use this cause no indent for Steps 1-8
+    inline = False # Defines whether content is to be displayed on same line
+    empty = False  # Defines whether tag is an empty tag
+    def __init__(self, content=None, **kwargs):
+        self.content_list = []
+        self.kwargs = kwargs
+        if content:
+            self.content_list.append(content)
+        
+    # Append content
+    def append(self, content):
+        self.content_list.append(content)
+        
+    # Start tag for element
+    def starttag(self, tag, indent):
+        if tag:
+            tag = self.indent_size * indent + '<' + tag
+            if self.kwargs:
```

Comment:
no need for this check, your for loop will iterate over empty dict and nothing will happen